website/code_issues.qmd

```diff
@@ -357,6 +357,30 @@ Ideally, use `requireNamespace("pkg")` or `require("pkg")`, both return `FALSE`
 
 
 
+--------------
+
+
+# Setting `options(warn = -1)`
+
+## Problem
+
+You are setting `options(warn = -1)`.
+
+## Solution
+
+Omit the code line and rather use `suppressWarnings()` if really needed.
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
+
+You are setting `options(warn=-1)` in your function. This is not allowed. Please rather use `suppressWarnings()` if really needed.
+
+:::
+
+CRAN doesn't allow negative `warn` options. This setting will turn off all warning messages. Even if the settings are correctly restored, as explained in the [Change of Options recipe](#change-of-options-graphical-parameters-and-working-directory), the submission will be rejected.
+
+Instead, CRAN recommends using `suppressWarnings()`. This will also disable warnings, it only affects the expression called within. 
```

Comment:
Small re-wording suggestion:

```suggestion
CRAN recommends using `suppressWarnings()`, which disables warnings only for the specific expression it's applied to, rather than globally.
```